scripts/ci-install-deps.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+### Install all dependencies on Ubuntu 16.04 LTS (Xenial Xerus) for RChain dev environment
+
+
+## Set BASH environment so it will fail properly throwing exit code
+set -euxo pipefail
+
+## setup tmp directory
+tmp_dir="/deps-tmp"
+if [ ! -d "${tmp_dir}" ]; then
+    mkdir ${tmp_dir}
+else
+    rm -rf ${tmp_dir}
+fi
+cd ${tmp_dir}
```

Comment:
I think what you want looks like this:

```
tmp_dir=$(mktemp -d ${TMPDIR:-/tmp}/ci-install-deps.sh.XXXXXX)
```

This creates the directory for you and will generate a new `tmp_dir` for each invocation. Is that what you wanted?